OWNERS

```diff
@@ -14,15 +14,11 @@ reviewers:
 - Boomatang
 - briangallagher
 - carlkyrillos
-- cecobask
```

Comment:
this commit needs to be removed, unrelated to your change

Reply:
Done. commit removed/reverted - https://github.com/integr8ly/cloud-resource-operator/pull/798/commits/412bf9b2c08ee69c3af62f5b0cb03dc3115bf5ba
Thank you